test/e2e/data/infrastructure-docker/cluster-template-ci.yaml

```diff
@@ -105,3 +105,19 @@ spec:
         name: "${ CLUSTER_NAME }-md-0"
         apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
         kind: DockerMachineTemplate
+---
+
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: MachineHealthCheck
+metadata:
+  name: "${CLUSTER_NAME}-mhc-0"
+spec:
+  clusterName: "${ CLUSTER_NAME }"
+  maxUnhealthy: 40%
+  selector:
+    matchLabels:
```

Comment:
IMO we should add a label to the MachineDeployment (something like nodepool=pool1), and then use the same labels here, so we can avoid to patch back the machines during the test